opendebates/management/commands/create_site.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib.sites.models import Site
+from django.core.management.base import LabelCommand
+
+
+class Command(LabelCommand):
+    def handle(self, *labels, **options):
+        for label in labels:
+            print(label)
+            Site.objects.get_or_create(name=label, domain=label)
```

Comment:
As mentioned in chat, this doesn't actually create a working site, afaik. You need to create it via the admin, where all the pre-populated values fill in reasonable defaults for you.

Reply:
This just creates a Django Site object. A user would still need to create Debates via the admin.

Reply:
Whoops! My bad. You're of course entirely correct.